shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -28,62 +29,141 @@
 import io.flutter.view.FlutterMain;
 
 /**
- * {@code Activity} which displays a {@link io.flutter.view.FlutterView} that takes up all
- * available space.
+ * {@code Activity} which displays a {@link FlutterFragment} that takes up all available space.
  *
  * {@link FlutterActivity} is the simplest and most direct way to integrate Flutter within an
  * Android app.
- * TODO(mattcarroll): how exactly does FlutterActivity need to be leveraged to display Flutter content?
  *
- * If Flutter is needed in a location that cannot use an {@code Activity>}, consider using
+ * By default, {@link FlutterActivity} configures itself to do the following:
+ *  - no splash screen
```

Comment:
Is this syntax supported for Lists in Javadoc? I was only aware of `<li>`.

Reply:
I don't see any Javadoc tags that represent list items, and I think we should avoid HTML in our Javadocs.  I'd rather leave this with literal formatting.

Reply:
`<li>` is the only officially-sanctioned way of doing lists in Javadocs that I'm aware of. `-` will render as a big contiguous blob in the HTML docs on our website.